dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-3.0/io.openliberty.mpOpenTracing-3.0.feature

```diff
@@ -7,13 +7,32 @@ IBM-App-ForceRestart: install, \
 IBM-ShortName: mpOpenTracing-3.0
 Subsystem-Name: MicroProfile OpenTracing 3.0
 IBM-API-Package: \
-    org.eclipse.microprofile.opentracing; type="stable"
--features=io.openliberty.opentracing-3.0, \
-  io.openliberty.mpConfig-3.0, \
-  io.openliberty.org.eclipse.microprofile.opentracing-3.0, \
-  io.openliberty.mpCompatible-5.0
--bundles=\
-    io.openliberty.microprofile.opentracing.2.0.internal.jakarta; apiJar=false; location:="lib/"
+    org.eclipse.microprofile.opentracing; type="stable", \
+    io.opentracing;  type="third-party", \
+    io.opentracing.tag;  type="third-party", \
+    io.opentracing.propagation;  type="third-party", \
+    io.openliberty.opentracing.spi.tracer; type="ibm-spi"
+-features= \
+    io.openliberty.mpConfig-3.0, \
+    io.openliberty.restfulWS-3.0, \
+    io.openliberty.cdi-3.0, \
+    io.openliberty.org.eclipse.microprofile.opentracing-3.0, \
+    io.openliberty.mpCompatible-5.0
+-bundles= \
+    io.openliberty.microprofile.opentracing.2.0.internal.jakarta; apiJar=false; location:="lib/", \
+    com.ibm.ws.jaxrs.defaultexceptionmapper.jakarta, \
+    io.openliberty.opentracing.2.0.internal.jakarta, \
+    io.openliberty.opentracing.2.0.internal.cdi.jakarta, \
+    io.openliberty.io.opentracing.opentracing-util.0.33.0, \
+    io.openliberty.opentracing.2.0.thirdparty; location:="dev/api/third-party/,lib/"; mavenCoordinates="io.opentracing:opentracing-api:0.33.0", \
+    com.ibm.ws.microprofile.opentracing.jaeger, \
+    com.ibm.ws.microprofile.opentracing.jaeger.adapter, \
+    com.ibm.ws.microprofile.opentracing.jaeger.adapter.impl,\
+    io.openliberty.opentracing.3.0.internal.restfulws, \
+    io.openliberty.opentracing.2.0.spi; location:="dev/spi/ibm/,lib/"
```

Comment:
This was in `jars` before, now it's in `bundles`. What's the difference?

Reply:
The jar in question was actually an OSGi bundle, that is why it moved. While we have observed that the spi package is also exported by `io.openliberty.opentracing.2.0.internal`, that should not be a problem for now. In the long run, the spi package should be exported solely by the spi bundle.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/19290